src/app/component/MenuOverlay/MenuOverlay.style.scss

```diff
@@ -233,7 +233,7 @@
 
     &-ItemCaption {
         font-size: 1.5rem;
-        line-height: 1;
+        line-height: 3.2rem;
```

Comment:
Remove this, implement via min-height. The line height you provided makes menu never wrap (if text will be long). Can choose a adequate line-height line 1.4